config/config.go

```diff
@@ -1259,6 +1275,9 @@ func SetServerDefaults(v *viper.Viper) error {
 		v.SetDefault("Federation_DirectorUrl", v.GetString(param.Server_ExternalWebUrl.GetName()))
 	}
 
+	// Warn the user if they still use IssuerKey
+	warnDeprecatedConfigs(v)
+
```

Comment:
See previous comment -- we should already be printing out a warning when `IssuerKey` is set in configuration.